common/lib/ucdc_edx_api_plugin/ucdc_edx_api/api/serializers.py

```diff
@@ -0,0 +1,97 @@
+from uuid import uuid4
+
+from django.contrib.auth.models import User
+from rest_framework import serializers
+from social_django.models import UserSocialAuth
+
+from openedx.core.djangoapps.user_api.accounts import (
+    EMAIL_CONFLICT_MSG,
+    USERNAME_CONFLICT_MSG,
+)
+from openedx.core.djangoapps.user_api.accounts.api import check_account_exists
+from openedx.core.djangoapps.user_api.accounts.api import (
+    get_email_validation_error,
+    get_username_validation_error,
+)
+from ucdc_edx_api.api.exceptions import ResourceConflicts
+
+
+class UserRegisterSerializer(serializers.ModelSerializer):
+    auth_uid = serializers.SerializerMethodField(read_only=True)
+
+    class Meta:
+        model = User
+        fields = ["id", "username", "email", "auth_uid"]
+
+    @staticmethod
+    def get_auth_uid(obj):
+        qs = UserSocialAuth.objects.filter(user_id=obj.id)
+        if qs.count() == 1:
```

Comment:
Please pay attention to this part of the code.

Reply:
done